### PR TITLE
Allow to suppress testng checks

### PR DESCRIPTION
**Pull request by kokosing**
Allow to suppress testng checks

### CODE REVIEW

File: presto-testng-services/src/main/java/io/prestosql/testng/services/SuppressTestngChecks.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Retention(RUNTIME)
+@Target(METHOD)
+public @interface SuppressTestngChecks
```
Comment (findepi) [inline presto-testng-services/src/main/java/io/prestosql/testng/services/SuppressTestngChecks.java:24]: Maybe name it to reflect it suppresses method annotation checks (and not eg test class naming)

Comment (kokosing) [reply presto-testng-services/src/main/java/io/prestosql/testng/services/SuppressTestngChecks.java:24]: What do you suggest? I was thinking about that I didn't find anything better? `@SuppressNonTestMethodCheck`?

Comment (findepi) [reply presto-testng-services/src/main/java/io/prestosql/testng/services/SuppressTestngChecks.java:24]: Maybe just `@ReportUnannotatedMethods.Suppress` (with the intention not to import that class directly) ?

Comment (kokosing) [reply presto-testng-services/src/main/java/io/prestosql/testng/services/SuppressTestngChecks.java]: Good idea, thanks.

**Review by losipiuk — APPROVED** [review]
LGTM, though I have doubts if this `ReportUnannotatedMethods` does not give more pain than benefit.

**Comment by kokosing** [comment]
> though I have doubts if this ReportUnannotatedMethods does not give more pain than benefit.

It is painful. But it provides benefits too. Imagine that we used to have test method without `@Test` annotation (false test coverage impression). Now it is not possible.

**Comment by losipiuk** [comment]
> > though I have doubts if this ReportUnannotatedMethods does not give more pain than benefit.
> 
> It is painful. But it provides benefits too. Imagine that we used to have test method without `@Test` annotation (false test coverage impression). Now it is not possible.

Yeah - I know what it does. Yet the same argument works for unit/integration tests. And we do not have mechanism there. And we assume we are good.

**Comment by kokosing** [comment]
> Yet the same argument works for unit/integration tests. And we do not have mechanism there.

This works for all tests, unit and product tests.

**Comment by losipiuk** [comment]
> > Yet the same argument works for unit/integration tests. And we do not have mechanism there.
> 
> This works for all tests, unit and product tests.

Oh. Good to know. I though it is only enabled for PTs.

**Pull request merged**